CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+cmake_minimum_required(VERSION 3.12..4.0)
```

Comment:
CMake added [ranges for required version in 3.12](https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html), that's why I choose this specific version